libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -424,6 +447,17 @@ private static Stream<InstrumentationService.InstrumentationInfo> pathChecks() {
         });
     }
 
+    private static void ensureClassesSensitiveToVerificationAreInitialized() {
```

Comment:
This seems odd. The class is only relevant because we had a bug, right? There's nothing inherently special about this class, so I don't see why we would need to force load it, especially in the case we've already got a VerifyError.